Apps/Medication/src/Controllers/MedicationStatementController.cs

```diff
@@ -75,38 +75,26 @@ public MedicationStatementController(IAuthorizationService authorizationService,
         [Produces("application/json")]
         [Route("{hdid}")]
         [Authorize(Policy = "PatientOnly")]
-        public async Task<ActionResult> GetMedicationStatements(string hdid, [FromHeader] string protectiveWord = null)
+        public async Task<IActionResult> GetMedicationStatements(string hdid, [FromHeader] string protectiveWord = null)
         {
             ClaimsPrincipal user = this.httpContextAccessor.HttpContext.User;
             var isAuthorized = await this.authorizationService.AuthorizeAsync(user, hdid, PolicyNameConstants.UserIsPatient).ConfigureAwait(true);
             if (!isAuthorized.Succeeded)
             {
-                return new ChallengeResult();
+               return new ForbidResult();
             }
-
+            RequestResult<List<MedicationStatement>> result = new RequestResult<List<MedicationStatement>>();
```

Comment:
nit lets inline the initialization of ResultStatus and ResultMessage

Reply:
Fixed